tools/mock-service-host/src/mid/oav/swaggerMocker.ts

```diff
@@ -58,33 +52,94 @@ export default class SwaggerMocker {
         }
         example.parameters = this.mockRequest(example.parameters, specItem.content.parameters, rp)
         example.responses = this.mockResponse(example.responses, specItem)
-        this.patchResourceIdAndType(example.responses, liveRequest)
+    }
+
+    public patchExampleResponses(
```

Comment:
Same with the previous concern. I wonder if it will hide the problem of the example. Is this initiative from that .Net will handle the resource hierarchically?

Reply:
not for hierarchically design, but it's because .net has logic check on "id"/"resource-type" values.  For example "/subscriptions/{subscriptionId}/resourceGroups/xxx" in not valid since "{subscriptionId}" is not in GUID format. I find too many this kind of case in example files, so tend to use patch first, and then we can talk with swagger team whether these can get fixed